### PR TITLE
Resolved issue#21 and a syntax error

### PR DESCRIPTION
**Pull request by KomalKalyan**
Please go through the changes i have made

### CODE REVIEW

File: connect_game.py [hunk 1]
```diff
@@ -93,7 +93,7 @@ def update(self):
         Checks the game state, dispatching events as needed.
         """
         if self.game_data.game_board.tie_move():
-            bus.emit("game:over", GameOver(was_tie=True))
+            bus.emit("game:over", self.renderer, GameOver(was_tie=True)) #There was a syntax error in this line
```
Comment (bymayanksingh) [inline connect_game.py]: please remove the one line comment.

**Review by bymayanksingh — CHANGES_REQUESTED** [review]

File: connect_game.py (hunk 1 above)
Comment (KomalKalyan) [reply connect_game.py]: should i remove all the comments in it or only that one in line96
i have comment even while fixing the issue#21

File: connect_game.py [hunk 2]
```diff
@@ -60,19 +60,23 @@ def mouse_click(self, event: MouseClickEvent):
...
-        pygame.display.update()
+            pygame.display.update()
 
-        self.game_data.turn += 1
-        self.game_data.turn = self.game_data.turn % 2
+            self.game_data.turn += 1
+            self.game_data.turn = self.game_data.turn % 2
+
+            #Solved the BUG when a column gets filled issue #21
+            #By pushing the latter part of this function into the "if" condition
```
Comment (bymayanksingh) [inline connect_game.py]: thanks, but please remove these comments too :)

**Review by bymayanksingh — CHANGES_REQUESTED** [review]

File: connect_game.py (hunk 1 above)
Comment (bymayanksingh) [reply connect_game.py]: yes, remove those too.

**Comment by bymayanksingh** [comment]
after you are done removing the comments, run black on the project. (follow readme on black usage)
then we are good to go with your PR

File: connect_game.py (hunk 1 above)
Comment (KomalKalyan) [reply connect_game.py]: ok 2 minutes

**Comment by KomalKalyan** [comment]
I have removed all my comments and i will run it in black and send you the PR, thanks!

**Comment by bymayanksingh** [comment]
Please check your slack message.
Merging this PR.
Thanks :)

**Comment by bymayanksingh** [comment]
Fixes #21

**Pull request merged**